packages/medusa/src/api/admin/returns/route.ts

```diff
@@ -41,11 +41,25 @@ export const POST = async (
   res: MedusaResponse
 ) => {
   const input = req.validatedBody as AdminPostReturnsReqSchemaType
```

Comment:
self-review: Move `location_id` from request items API to this one. We only allow creating a return for a single location at a time